frontend/src/context/mapStateSlice/selectors.ts

```diff
@@ -5,11 +5,15 @@ import { Map as MapBoxMap } from 'mapbox-gl';
 import type { RootState } from '../store';
 import type { LayerDataTypes } from '../layers/layer-data';
 import type { MapState } from '.';
-import type { LayerKey } from '../../config/types';
+import type { LayerKey, LayerType } from '../../config/types';
 import { BoundaryRelationsDict } from '../../components/Common/BoundaryDropdown/utils';
 
 export const layersSelector = (state: RootState): MapState['layers'] =>
   state.mapState.layers;
+export const activeLayersSelector = (state: RootState): MapState['layers'] =>
+  state.mapState.layers.filter((layer: LayerType) => {
+    return !layer.id.includes('boundaries');
```

Comment:
you can use `layer.type !== 'boundary'` instead which is safer

Reply:
Of course.